config/ground-truth-yaml-files/all_features_version_10_no_visit_variant_dili.yml

```diff
@@ -529,45 +577,49 @@ patient:
     categories:
     - biolink:PhenotypicFeature
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id114
+    enum:
     - '0'
     - '1'
     type: string
     search_term: eosinophilia
+    limit: 5
   EthnicityDILI:
     categories:
     - biolink:PhenotypicFeature
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id126
+    enum:
     - '0'
     - '1'
     type: string
-    search_term: ethnicity
+    search_term: racial group
+    limit: 4
   FamilyHistoryLiverDisease:
     categories:
     - biolink:PhenotypicFeature
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id128
+    enum:
     - '0'
     - '1'
     type: string
-    search_term: family history liver disease
+    search_term: liver disease, family history 
+    limit: 1

```

Comment:
Should two search terms be separated similarly to what are on lines 557-560 above? I think making how to specify multiple search terms consistently is good.

Reply:
I agree with Hong. Search term consistency is a must. Is this two search terms, or do I send this entire string to the name resolver? In my mind, each feature should only have one search term.

Reply:
So, this is actually one search term: "liver disease, family history".  However, I just tested it without the comma, and Name Resolver finds the correct CURIE, so I can remove the comma (done). Also, in certain cases (e.g., registry data), one feature is actually two features combined, e.g., PrednisoneOrCorticosteroids_Rx. Because we did not collect the data, we cannot split the variable into two separate ones. I handled this as follows:

```
    search_term: prednisone
    limit: 1
    search_term: corticosteroid
    limit: 1
```

Does this work?

Reply:
In this case, only corticosteroid search_term will be used since it will override the previously defined search term prednisone when the yml file is loaded into a JSON dictionary.

Reply:
That's the equivalent of a list of objects in yaml

Reply:
I see. Done!